internal/server/routes/common/containers.go

```diff
@@ -2,6 +2,7 @@ package common
 
 import (
 	"fmt"
+	"github.com/joyrex2001/kubedock/internal/model/types"
```

Comment:
I order the imports in sets of standard library imports, thirdparty imports and last block local imports. The import will change (considering the remark about the types package), but if another import is used, can you use that pattern for imports as well?